packs.json

```diff
@@ -476,6 +476,15 @@
         "position": 1,
         "size": 20
     },
+    {
+        "code": "win",
+        "cycle_code": "kitara",
+        "date_release": null,
+        "ffg_id": 54,
+        "name": "Whispers in Nalubaale",
+        "position": 5,
+        "size": 20
+    }
```

Comment:
There's a missing comma after the closing brace. Please make sure to run the `validate.py` script before committing to avoid errors like this.

Reply:
Ok, will do that in the future!